Lowers Vulpkanin Heat Vulnerability

CawsForConcern opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
This changes Vulpkanin heat / cold resistances to match upstream Vulpkanin
- 50% extra heat damage -> 15% extra heat damage
- 50% less cold damage -> 15% less cold damage

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
Vulpkanin have been merged with upstream, and the numbers are much more mild there (15% extra heat / 15% less cold)
- https://github.com/space-wizards/space-station-14/pull/37539

Relevant Discord suggestions threads: 
- https://discord.com/channels/1272545509562777621/1412093233051467856
- https://discord.com/channels/1272545509562777621/1414236045725401108

Laser carbines became a lot more relevant after their buffs, going from 12 damage base to 20 damage base and having far more shots before needing to recharge. The HOS has had a new heat-dealing gun for a while, the Proto-5x energy gun, and the Warden inherited the HOS' old energy shotgun. There's just a lotta heat damage flying around on stations nowadays.

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="578" height="628" alt="image" src="https://github.com/user-attachments/assets/9052f9ed-3eea-43ac-950d-2dba4cda8c63" />


## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.
-->
:cl: CawsForConcern
- tweak: Lowered Vulpkanin heat damage modifier to 1.15 (15% extra heat) from 1.5 (50% extra heat)
- tweak: Lowered Vulpkanin cold damage modifier to 0.85 (15% less cold) from 1.5 (50% less cold)
- tweak: Updated Guidebook to reflect the new damage modifier
- tweak: Clarified in the Guidebook that Vulpkanin take extra Heat damage, not 'fire' damage

CawsForConcern committed:
- tweak: Lowered Vulpkanin heat damage modifer to 1.3 (30%)
- tweak: Updated Guidebook to reflect the new damage modifer

CawsForConcern committed:
- tweak: Clarified in the Guidebook that Vulpkanin take extra Heat damage, not 'fire' damage

CawsForConcern commented:
oh boy Vulps are coming to upstream and they have even milder numbers:
- https://github.com/space-wizards/space-station-14/pull/37539

I might drop the numbers on this PR to match upstream, but I'll wait for more feedback / input before I do.

CawsForConcern committed:
Upstream Vulpkanin numbers - 15% extra heat / 15% less cold

Dimyell commented:
Starlight has a far more obnoxious heat meta than upstream, where vulps still only have a slight heat weakness. It'd feel a lot better to copy upstreams numbers on this to at least take away one of our vulps major downsides.

CawsForConcern commented:
Changed the numbers to match upstream Vulpkanin (15% extra heat / 15% less cold), updated media, updated changelogs.

Other additions like the MessyDrinker tag will have to wait for upstream merges

Fasuh commented:
Personally i think this is fine, 50% seems to be very bad, alternatively make them get 150% heat from atmos, but only 115% from heat weapons
